app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use App\Models\Posts;
+use Illuminate\Http\RedirectResponse;
+use App\Contracts\Services\UserServiceInterface;
+use Illuminate\View\View;
+use Illuminate\Support\Facades\Auth;
+
+class UserController extends Controller
+{
+    protected $userService;
+    protected $postService;
+
+    public function __construct(UserServiceInterface $userService)
+    {
```

Comment:
How about this one too?
https://github.com/ShweLinLaeKyaw69/laravelproject/pull/1/files#r1637923959